yarn.lock

```diff
@@ -2028,6 +2038,19 @@ ethereumjs-util@^6.0.0:
     rlp "^2.2.3"
     secp256k1 "^3.0.1"
 
+ethereumjs-util@^6.2.1:
```

Comment:
Nit: There's a compatible range version of this immediately above this. If you delete the above entry and re-run `yarn`, it should combine them.

Reply:
tried what you recommended here and didn't see any change 🤔

Reply:
What did you try? By "delete the above entry" I meant delete lines 2028-2039. The dedupe was successful for me locally.

Reply:
ah I did mis-read that. Thought you meant this entry (starting line 2041). Worked this time.